+++ /usr/local/bundle/gems/govuk_tech_docs-2.0.12/lib/assets/javascripts/_modules/search.js
@@ -34,7 +34,7 @@
     this.downloadSearchIndex = function downloadSearchIndex () {
       updateTitle('Loading search index')
       $.ajax({
-        url: '/search.json',
+        url: '<%= ENV["ROOT_DOCPATH"] %>/search.json',
         cache: true,
         method: 'GET',
         success: function (data) {
